src/Menu/MenuItem.js

```diff
@@ -0,0 +1,31 @@
+import React, { PropTypes } from 'react';
+import Link from 'react-router/lib/Link';
+import styles from './style.scss';
+
+const MenuItem = ({ children, icon, href, onClick }) => {
+  const Type = href ? Link : 'a';
+
+  return (
+    <li className={styles.item}>
```

Comment:
can we allow users to pass their own `classNames`? so minor extensions and adaptation can happen?